app/com/linkedin/drelephant/tez/fetchers/TezFetcher.java

```diff
@@ -93,15 +98,17 @@ public TezApplicationData fetchData(AnalyticJob analyticJob) throws IOException,
 
           List<TezTaskData> mapperList = new ArrayList<TezTaskData>();
           List<TezTaskData> reducerList = new ArrayList<TezTaskData>();
+          List<TezTaskData> scopeTaskList = new ArrayList<TezTaskData>();
 
           // Fetch task data
           URL vertexListUrl = _urlFactory.getVertexListURL(dagId);
-          _jsonFactory.getTaskDataAll(vertexListUrl, dagId, mapperList, reducerList);
+          _jsonFactory.getTaskDataAll(vertexListUrl, dagId, mapperList, reducerList, scopeTaskList);
 
-          if(mapperList.size() + reducerList.size() > maxSize){
+          if(mapperList.size() + reducerList.size()+scopeTaskList.size() > maxSize){
```

Comment:
Add a space between second "+" and scopeTaskList?

Reply:
Will add the space for readability